gradle/publishing.gradle

```diff
@@ -74,17 +73,6 @@ if (ext.publishLibrary) {
             }
         }
     }
-    artifactoryPublish {
```

Comment:
This does mean that `build.py` will break, but we're not using that anywhere, and plan to remove it very shortly, so that seems fine.